package com.sqli.intern.api.validator.utilities.exceptions;public class TestRunException {
}
